src/librbd/deep_copy/ImageCopyRequest.cc

```diff
@@ -14,7 +14,8 @@
 #define dout_subsys ceph_subsys_rbd
 #undef dout_prefix
 #define dout_prefix *_dout << "librbd::deep_copy::ImageCopyRequest: " \
-                           << this << " " << __func__ << ": "
+                           << this << " " << __func__ << ": src_img_id=" \
+			    << m_src_image_ctx->id << " "
```

Comment:
I prefer an approach when in dout_prefix we output only `this`, while in the constructor (or create request) you log all variables you want. This way the log line is not very long (due to the same info repeated on every line), while you may use `this` address to grep all lines for the same request and see all params in the "create request" log message.

But this is just my preference. I will not mind if this is merged.

Reply:
Our comments raced ;)

Reply:
Using the "this" address is the existing log approach  but IMO that makes debugging quite difficult, especially in the rbd-mirror daemon when multiple images are being processed at the same time. It also makes it very difficult to track the operation flow as there is no connection between one class and the next. 
Having a common image id or object id makes it a lot easier to see the entire flow instead of tracking individual instances of a class.